panel/src/components/Forms/Blocks/Block.vue

```diff
@@ -202,6 +202,14 @@ export default {
         }
       }
     },
+    onFocusIn(event) {
+      // skip focus if the event is coming from the options buttons
```

Comment:
```suggestion
      // skip focus if the event is coming from the options buttons
      // to preserve the current focus (since options buttons directly
      // trigger events and don't need any focus themselves)
```